web/common.css

```diff
@@ -0,0 +1,5 @@
+body {
```

Comment:
maybe we could put the web/ folder to into the data repo? (so content guys can work on that)